src/main/scala/org/holmesprocessing/totem/services/cfg/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM golang:alpine
+
+# create folder
+RUN mkdir -p /service
+WORKDIR /service
+
+# get go dependencies
+RUN apk add --no-cache \
+		git \
+	&& go get github.com/julienschmidt/httprouter \
+	&& rm -rf /var/cache/apk/*
+
+###
+# CFG specific options
+###
+
+# Get Go CFG dependencies
+RUN apk add --no-cache \
+		bash \
+		build-base \
+		binutils \
+		binutils-dev \
+		capstone \
```

Comment:
Still some errors and warning from the Dockerfile. Make sure that it builds completely on your machine.